assembly/src/assembler/mast_forest_builder.rs

```diff
@@ -569,3 +570,13 @@ fn should_merge(is_procedure: bool, num_op_batches: usize) -> bool {
         true
     }
 }
+
+impl MastForestBuilder {
```

Comment:
```suggestion

/// Error handling
impl MastForestBuilder {
```

Could you also move this up out of the `// HELPER FUNCTIONS` header?

Reply:
done